src/apb/cli.py

```diff
@@ -387,6 +388,56 @@ def subcmd_test_parser(subcmd):
     )
     return
 
+def subcmd_run_parser(subcmd):
+    """ provision subcommand """
+    subcmd.add_argument(
+        '--project',
+        action='store',
+        dest='project',
+        required=True,
+        help=u'Project where the APB should be run'
+    )
+
+    subcmd.add_argument(
+        '--action',
+        action='store',
+        dest='action',
+        required=False,
+        help=u'The action to perform when running the APB',
+        default='provision'
+    )
+
+    subcmd.add_argument(
+        '--tag',
+        action='store',
+        dest='tag',
+        help=u'Tag of APB to build'
+    )
+
+    subcmd.add_argument(
+        '--registry',
```

Comment:
Do we need this if we are already prompting for `tag`?

Reply:
I noticed we have the same thing for build... Maybe it's okay. Just feels weird. I wouldn't be opposed to removing it from build either.